packages/vscode-extension/src/commonlib/common/constant.ts

```diff
@@ -12,7 +12,7 @@ export const loggingIn = "LoggingIn";
 export const switching = "Switching";
 export const initializing = "Initializing";
 
-export const m365CacheName = "appStudio";
```

Comment:
Besides the vsc change, if user installs old ttk in new vsc, will that break CLI login?

Reply:
After deleting Keytar in VS Code, users who are using the old VS Code extension and CLI may encounter these scenarios:
1. Login in the CLI -> CLI commands work fine -> User need to login in the VS Code extension -> if account is the same -> all work fine in VS Code extension and CLI

2. Login in the VS Code extension -> VS Code extension works fine -> User needs to login in the CLI -> if account is the same -> all work fine in the VS Code extension and the CLI

3. Login in the CLI -> CLI commands work fine -> User need to login in the VS Code extension -> if account is different -> CLI login info will be clear out, the user needs to login in the CLI again

4. Login in the VS Code extension -> VS Code extension works fine -> User needs to login in the CLI -> if account is different -> VS Code extension login info will be clear out after reloading, the user needs to login in CLI again (If the user does not reload VS Code, all teamsfx commands in VS Code extension work fine)

For users who are using the new VS Code extension and CLI.
Everything works fine because the account and token cache are the same in different files.